test/Kestrel.Core.Tests/HttpUtilitiesTest.cs

```diff
@@ -170,7 +170,8 @@ public static TheoryData<string> HostHeaderData
         [MemberData(nameof(HostHeaderData))]
         public void ValidHostHeadersParsed(string host)
         {
-            Assert.True(HttpUtilities.IsValidHostHeader(host));
+            HttpUtilities.ValidateHostHeader(host);
+            Assert.True(true);
```

Comment:
`Assert.True(true)`... 😄 xunit used to have Assert.DoesNotThrow 😄

Reply:
Wasn't sure whether to just put nothing in as if it threw it would catch it as an error; but that felt wrong 😄

Reply:
I'd remove it and just add a comment saying this shouldn't throw.

Reply:
Fixed